### PR TITLE
taxonomy: italien ingredient

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**
### Related issue(s) and discussion

- Fixes #[ISSUE NUMBER]

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -35006,13 +35006,13 @@ ciqual_food_name:en:Apricot, pitted, dried
 ciqual_food_name:fr:Abricot, dénoyauté, sec
 #24 in 3 @2021-09-12
 
-<en:apricot
+
```
Comment (alexgarel) [inline taxonomies/ingredients.txt:35009]:
```suggestion
:+1: for removing apricot.

Shouldn't we add a parent ? like the one of almonds (en:tree nut) ?
```

**Review by alexgarel — APPROVED** [review]
LGTM

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (moon-rabbitOFF) [reply taxonomies/ingredients.txt:35009]: I put it in tree nut, thank you

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -35006,13 +35006,13 @@ ciqual_food_name:en:Apricot, pitted, dried
...
 
-<en:apricot
+<en:tree nut
 en:apricot kernels
 de:Aprikosenkerne
 es:huesos de albaricoque
 fi:aprikoosinsiemen, aprikoosinsiemenet
 fr:noyaux d'abricots, noyau d'abricot
-it:Noccioli di Albicocca
+it:Noccioli di albicocca, mandorle d'albicocca
```
Comment (VaiTon) [inline taxonomies/ingredients.txt:35015]:
```suggestion
it:Armelline, Noccioli di albicocca, mandorle d'albicocca
```

See https://it.wikipedia.org/wiki/Armellina

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**